apps/full-site-editing/full-site-editing-plugin/starter-page-templates/page-template-modal/index.js

```diff
@@ -55,10 +55,10 @@ class PageTemplateModal extends Component {
 		this.props.insertTemplate( title, previewBlocks );
 	};
 
-	selectTemplate = () =>
+	handleConfirmation = () =>
 		this.setTemplate( this.state.slug, this.state.title, this.state.previewBlocks );
 
-	focusTemplate = ( slug, title, previewBlocks ) => {
+	previewTemplate = ( slug, title, previewBlocks ) => {
```

Comment:
👍 for this change, but you will need to update the call as well.

```es6
onTemplateSelect={ this. previewTemplate }
```

Reply:
Can you point me where? I'm only able to locate a single usage and I've already updated it as part of my changes here.

Reply:
You already did it. Have you rebased? A few minutes ago the change below wasn't there:

![image](https://user-images.githubusercontent.com/77539/63293634-e7239180-c29e-11e9-8ad0-d379cdc88390.png)

Reply:
Maybe a bad review, though.

Reply:
I haven't touched anything since I made the PR.